frontend/src/util/firebase/chain.ts

```diff
@@ -43,6 +40,12 @@ const getChains = async () => {
   );
 };
 
+const getChainsLength = async () => {
+  const snapshot = await db.collection("chains").get();
+
+  return snapshot.docs.length;
```

Comment:
Given chains are fetched in multiple parts of the application, it might be a good idea to create a context that shares that data once. Doesn't need to happen now though.